### PR TITLE
Refactor verify-email to latest design.

### PR DESCRIPTION
**Pull request by Ddeak**
## Description
This commit updates the verify-email to the latest designs. In doing so, the new designs require the app name and logo to be sent with the e-mail, so a minor re-factor is done to pass those veriables.

## Source
https://dswift.atlassian.net/browse/DAS-613

## Type
- [ ] Bug Fix
- [ ] Feature Addition
- [x] Refactor
- [ ] HotFix

## Checklist
- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Review by xavji — DISMISSED** [review]

File: hat/app/org/hatdex/hat/utils/Mailer.scala [hunk 1]
```diff
@@ -145,96 +147,90 @@ class HatMailerImpl @Inject() (
   def serverErrorNotify(
       request: RequestHeader,
       exception: UsefulException
-    )(implicit m: Messages): Future[Unit] = {
+    )(implicit m: Messages): Future[Unit] =
```
Comment (whiteshadow-gr) [inline hat/app/org/hatdex/hat/utils/Mailer.scala:150]: @xavji is the removal of the `{` expected? Just checking to not add noise to the future PRs

**Comment by whiteshadow-gr** [comment]
@Ddeak you can remove the **custom** email code now that we merged the 2 in 1. Both the HTML and text.

File: hat/app/org/hatdex/hat/utils/Mailer.scala (hunk 1 above)
Comment (xavji) [reply hat/app/org/hatdex/hat/utils/Mailer.scala:150]: it is `scalafmt`, yes it is all good

**Review by whiteshadow-gr — APPROVED** [review]
Just delete the custom email templates, please. No need to have it if it's not going to be used.

**Review by whiteshadow-gr — APPROVED** [review]

**Pull request merged**